FixMyBerlin/fixmy.platform
# Explicitly set permit duration per Gastro application

Gastro permit applications have an implicitly defined permit start and end date. This PR adds a model fields which allows setting permit start and end on a per-application level. The value is set on migration and when sending a notice for an accepted application depending on the campaign that an application is associated with.

Part of #401 

## Type of change

Please delete options that are not relevant.

- New feature (non-breaking change which adds functionality)

## How Has This Been Tested?

- Set an application as accepted and send a notice, permit start and end times have been set
- Apply db migrations and previously accepted applications have their permit start and end date set

## Checklist:
